src/libraries/System.Runtime/tests/TrimmingTests/AppDomainGetThreadGenericPrincipalTest.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using System.Security.Principal;
+using System.Threading;
+using System.Diagnostics;
+
+class Program
+{
+    static int Main(string[] args)
+    {
+    	// Thread.CurrentPrincipal calls AppDomain.CurrentDomain.GetThreadPrincipal() which
```

Comment:
(nit) whitespace is mis-aligned here and in AppDomainGetThreadWindowsPrincipalTest.cs